src/thor/dijkstras.cc

```diff
@@ -163,7 +157,7 @@ void Dijkstras::ExpandForward(GraphReader& graphreader,
     // Let implementing class we are expanding from here
     EdgeLabel* prev_pred =
         pred.predecessor() == kInvalidLabel ? nullptr : &bdedgelabels_[pred.predecessor()];
-    ExpandingNode(graphreader, pred, tile->get_node_ll(node), prev_pred);
+    ExpandingNode(graphreader, tile, nodeinfo, pred, prev_pred);
```

Comment:
this was previously just the LL with the idea it was used in isochrones however its more generic to pass the actual node and tile and predecessors for other derived class' use-cases